Formula/makepkg.rb

```diff
@@ -14,14 +14,12 @@ class Makepkg < Formula
     sha256 high_sierra: "c8f2f6999669c56b5e40e2608ad1e0adfe2c8eb73f8cef959a229856d21da6ed"
   end
 
-  disable! date: "2022-03-28", because: "depends on fakeroot which does not build"
-
   depends_on "asciidoc" => :build
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
   depends_on "docbook-xsl" => :build
-  depends_on "libtool" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build
   depends_on "bash"
   depends_on "fakeroot"
   depends_on "gettext"
```

Comment:
Is this needed on macOS?

Reply:
I just tried with `uses_from_macos "python" => :build` instead and it works, so it does not appear to need a specific version of Python 3 to build on macOS.  I will change this in the next push.

Reply:
I was actually referring to gettext, since we pass `i18n=false`.

Reply:
I dropped the `gettext` dependency on macOS and it worked fine locally, so it is now a Linux-only dependency.